frontend/src/search/hooks/useSearch.ts

```diff
@@ -0,0 +1,107 @@
+import { useState, useEffect } from 'react';
+import { useSearchFoldersQuery, useSearchJournalsQuery } from '@/graphql/generated';
+
+const useSearch = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [isSearching, setIsSearching] = useState(false);
```

Comment:
It doesn't seem that you're using isSearching anywhere — can remove it

Reply:
Done!